packages/oml/oml.0.0.6/opam

```diff
@@ -12,7 +12,13 @@ remove: ["ocamlfind" "remove" "oml"]
 depends: [
   "ocamlfind" {build}
   "cppo" { build & >= "1.4.0" }
-  "lacaml" { >= "8.0.6" }
-  "lbfgs" { >= "0.8.7" }
-  "ocephes" { >= "0.8" }
+  "lacaml"
+  "lbfgs"
+  "ocephes"
+]
+conflicts : [
+  "lacaml" { <= "8.0.6" }
```

Comment:
This isn't quite equivalent to the previous dependency -- did you mean `<"8.0.6"`